templates/case-study/nova-telecommunications.html

```diff
@@ -0,0 +1,255 @@
+    {%- if slot == 'image_content' %}
+      {{ image(url="https://assets.ubuntu.com/v1/e0db72d3-image1.jpg",
+            alt="",
+            width="2464",
+            height="1028",
+            hi_def=True,
+            loading="auto|lazy") | safe
+      }}
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) text(title="Keeping up the pace for a growing customer base") -%}
+    {%- if slot == 'content' -%}
+      <p>
+        In the face of rapid technological evolution, one of the constants has been the importance of telecommunications. Telecommunications systems embody technological evolution and bring it within reach of ordinary consumers. The journey from landline telephones to mobile services, 5G and superfast internet has been driven by telco operators who work to commercialize these innovative technologies. In order for telecommunications companies to keep supporting innovation, they need to be innovative with their underlying infrastructure.
+      </p>
+      <p>
+        Nova is a telecommunications operator that provides mobile, broadband, television and fixed network services to households and businesses in Greece. Recognizing the need for innovation, Nova intends to scale-up and scale-out the services it offers to customers, but wanted to make sure innovation started at home. In order to grow sustainably, improve time to market, and continue delivering high-quality connectivity to their customers, Nova identified Canonical as the partner to help them pivot to a fully open source private cloud infrastructure.
+      </p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) text(title="Challenge") -%}
+    {%- if slot == 'content' -%}
+      <p>
+        An additional complication was that Nova’s private cloud infrastructure was split into three separate clouds. These were managed in-house, with two sourced from proprietary vendors, and one open source cloud sourced from Canonical. While this was necessary from an operational perspective, the fragmentation of vendors brought additional complexity.
+      </p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) vf_quote_wrapper(
+    quote_text="We are committed to delivering the full spectrum of mobile and fixed telecommunication services to our customers, which will enable us to realize the greatest potential for growth and value creation. Through investments in our on-premises infrastructure, we are focusing on streamlining our operation to enhance our efficiency and productivity.",
+    citation_source_name_text="George Orfanidis",
+    citation_source_title_text="Core &amp; Services Director, Nova",
+    is_shallow=True
+    )
+    -%}
+  {%- endcall -%}
+
+  {%- call(slot) image_wrapper() -%}
+    {%- if slot == 'image_content' %}
+      {{ image(url="https://assets.ubuntu.com/v1/bceecbd7-image2.jpg",
+            alt="",
+            width="2464",
+            height="1028",
+            hi_def=True,
+            loading="auto|lazy") | safe
```

Comment:
For the rest of the images as well please
```suggestion
            loading="lazy") | safe
```